external-crates/move/crates/move-compiler/tests/move_2024/typing/index_underspecified.move

```diff
@@ -0,0 +1,14 @@
+module a::m;
```

Comment:
How is this test different from `index_invalid_crash`?

Reply:
The `T` used in table is underspecified, though the test output is the same.